compiler/src/main/java/com/squareup/anvil/compiler/codegen/ClassReferenceExtensions.kt

```diff
@@ -0,0 +1,52 @@
+package com.squareup.anvil.compiler.codegen
+
+import com.squareup.anvil.compiler.contributesMultibindingFqName
+import com.squareup.anvil.compiler.internal.reference.AnvilCompilationExceptionClassReference
+import com.squareup.anvil.compiler.internal.reference.ClassReference
+import com.squareup.anvil.compiler.internal.reference.Visibility
+import org.jetbrains.kotlin.name.FqName
+import org.jetbrains.kotlin.util.capitalizeDecapitalize.toLowerCaseAsciiOnly
```

Comment:
I've never seen this function before 😅  I think `"".lowercase()` is the new way in the std function (the other methods are deprecated).

Reply:
Lol same. Ahh nice, didn't even see that since I started `.toL` and it only showed the deprecated ones + the Ascii one. Updated now.